### PR TITLE
Change Email Templates for OCF and OSC

### PR DESCRIPTION
**Pull request by SudharakaP**
This is with regard to the following PR: https://github.com/opencollective/opencollective-api/pull/6007#issuecomment-860583619.

@alanna @znarf : Please take a look, I've made changes so that it mimics the above PR. But maybe it needs some tweaking.

### CODE REVIEW

**Comment by znarf** [comment]
Thank you @SudharakaP, waiting for @alanna's feedback first and will do a technical review when it's ready.

**Comment by SudharakaP** [comment]
> Thank you @SudharakaP, waiting for @alanna's feedback first and will do a technical review when it's ready.

@znarf : You are welcome. 

One other thing is that looking at, 

https://github.com/opencollective/opencollective-api/blob/d7360ba3df21dfddd88549cb49e877efda661026/server/lib/email.js#L296-L320

we don't seem to use, `thankyou.fearlesscitiesbrussels.hbs`, `thankyou.sfartandfilm.hbs` and `thankyou.sustainoss.hbs`. Okay to remove them? 🤔

**Comment by znarf** [comment]
@SudharakaP have a closer look at the code and you'll see that `sustainoss` can still be used. Ok to delete the others.

**Comment by SudharakaP** [comment]
> @SudharakaP have a closer look at the code and you'll see that `sustainoss` can still be used. Ok to delete the others.

@znarf : Ops, yes; my eye missed it. Sorry. 🙄 👁️ I've removed the other two.

**Comment by alanna** [comment]
@znarf sorry I find the whole interface of pull requests too confusing to navigate, so it's really hard for me to check the specific changes. But if you followed the pattern of the other email template changes, I am sure it's fine, so please go ahead.

**Comment by znarf** [comment]
@alanna I think it's important that you have a look to avoid back and forth, if the PR interface is confusing, simply check those files:

- https://github.com/opencollective/opencollective-api/blob/feat/changes-to-ocf-and-osc-email-templates/templates/emails/thankyou.opensource.hbs
- https://github.com/opencollective/opencollective-api/blob/feat/changes-to-ocf-and-osc-email-templates/templates/emails/thankyou.foundation.hbs

**Comment by znarf** [comment]
@SudharakaP the platformTipPdf and "Payment Details" seems to be missing from the `opensource` template.

File: templates/emails/thankyou.opensource.hbs [hunk 1]
```diff
@@ -16,16 +12,16 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
 
 {{#if interval}}
   {{#if firstPayment }}
-    <p>We would like to thank you for your pledge to give {{currency order.totalAmount currency=order.currency}} per {{interval}}.
-      Your first payment was received on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
+    <p>Thank you for contributing {{currency order.totalAmount currency=order.currency}} per {{interval}} to {{collective.name}}.
+       Your first payment was made on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
```
Comment (BenJam) [inline templates/emails/thankyou.opensource.hbs:16]:
```suggestion
       Your first payment was made on {{moment order.createdAt}}. As the collective's fiscal sponsor, Open Source Collective acknowledges your donation.</p>
```
Remove confusing note about Open Source Collective's tax status as users are confused about the implications on their own tax affairs.

File: templates/emails/thankyou.opensource.hbs [hunk 2]
```diff
@@ -16,16 +12,16 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
...
       {{> charge_date_notice}}
   {{else}}
-    <p>We would like to thank you for your continued support. Your latest {{interval}}ly contribution of {{currency order.totalAmount currency=order.currency}} was received on {{moment transaction.createdAt}}.
+    <p>Thank you for your continued support! Your latest {{interval}}ly contribution of {{currency order.totalAmount currency=order.currency}} was received on {{moment transaction.createdAt}}.
       As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
```
Comment (BenJam) [inline templates/emails/thankyou.opensource.hbs:20]:
```suggestion
      As the collective's fiscal sponsor, Open Source Collective acknowledges your donation.</p>
```

File: templates/emails/thankyou.opensource.hbs [hunk 3]
```diff
@@ -16,16 +12,16 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
...
   {{/if}}
 {{else}}
 
-<p>We would like to thank you for your contribution of {{currency order.totalAmount currency=order.currency}} on {{moment order.createdAt}}.
+<p>Thank you for your contribution of {{currency order.totalAmount currency=order.currency}} on {{moment order.createdAt}}.
   As the collective's fiscal sponsor, Open Source Collective, a public recognized tax-exempt organization under IRS Section 501(c)(6), acknowledges your donation.</p>
```
Comment (BenJam) [inline templates/emails/thankyou.opensource.hbs:25]:
```suggestion
  As the collective's fiscal sponsor, Open Source Collective acknowledges your donation.</p>
```

**Review by BenJam — CHANGES_REQUESTED** [review]
jumping into this PR with some mods as a result of [feedback](https://opencollective.freshdesk.com/a/tickets/22702) received in which users are confused about whether their contributions are tax deductable or not.

**Comment by BenJam** [comment]
@alanna I jumped into @SudharakaP's PR as he was touching the files anyway. @SudharakaP apologies if this is _not cool_ I am happy to submit a PR over the top of yours if you prefer. Ta!

**Comment by SudharakaP** [comment]
> @alanna I jumped into @SudharakaP's PR as he was touching the files anyway. @SudharakaP apologies if this is _not cool_ I am happy to submit a PR over the top of yours if you prefer. Ta!

@BenJam : Thanks a lot for the modifications; no, it's totally fine. 😄 😄 😄

**Comment by SudharakaP** [comment]
> @SudharakaP the platformTipPdf and "Payment Details" seems to be missing from the `opensource` template.

@znarf : Thanks for pointing it out. I've added that as well in this PR. 👍🏽

File: templates/emails/thankyou.opensource.hbs [hunk 4]
```diff
@@ -34,24 +30,40 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
...
 <h2>Help us raise more money!</h2>
-<p>Share this URL: <a href="https://opencollective.com/{{collective.slug}}/donate">https://opencollective.com/{{collective.slug}}/donate</a>
+<p>See our activities and transparent budget on
+  <a href="https://opencollective.com/{{collective.slug}}/donate">https://opencollective.com/{{collective.slug}}/donate</a>, and share the link to help us spread the word!
 </p>
 
 {{#if transactionPdf}}
 <h2>📎 Attachments</h2>
 <ul>
   <li>A PDF receipt of your transaction</li>
```
Comment (znarf) [inline templates/emails/thankyou.opensource.hbs:40]: <li>A PDF receipt of your contribution to {{collective.name}}</li>

**Comment by znarf** [comment]
> > @SudharakaP the platformTipPdf and "Payment Details" seems to be missing from the `opensource` template.
> 
> @znarf : Thanks for pointing it out. I've added that as well in this PR. 👍🏽

They seems to be missing from the `foundation` template too.

File: templates/emails/thankyou.opensource.hbs (hunk 4 above)
Comment (SudharakaP) [reply templates/emails/thankyou.opensource.hbs:40]: Thanks. Correct that in both templates. 😄

**Comment by SudharakaP** [comment]
> > > @SudharakaP the platformTipPdf and "Payment Details" seems to be missing from the `opensource` template.

> They seems to be missing from the `foundation` template too.

@znarf : Yes I've added the platformTipPdf and Payment Details section to the foundation email template as well. 😄

**Comment by BenJam** [comment]
would love to close this out if we have the time @SudharakaP

**Comment by SudharakaP** [comment]
@BenJam : Sure, I normally wait for one review on the PR at least to merge it. 

@kewitz : Could you review it by any chance since both Francois and Ben are out. Now I've corrected some issues Francois mentioned, but let me know if there's anything else to be done. 😄

**Comment by SudharakaP** [comment]
As per https://opencollective.slack.com/archives/C0RMV6F8C/p1626093851225900 I am merging this. 😄

**Pull request merged**